locale/requirements.txt

```diff
@@ -0,0 +1,3 @@
+dennis==1.1.0 \
```

Comment:
Why is this in this PR ? I don't see it being required anywhere. Also, requirements files are expected to live under `requirements/` top level directory to be picked up by our (current) [dependabot config](https://github.com/mozilla/addons-server/blob/master/.github/dependabot.yml).

Reply:
You are so very right. Somehow at some point I was having the compile-mo.sh script failing and so I thought I had to install dennis, but that makes no sense, it's already in prod.txt so I removed this.

Build: https://github.com/mozilla/addons-server/actions/runs/8055526462/job/22002582564?pr=21914